src/ProjectNeon/Assets/Data/Templates/Team.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Team : Target

```

Comment:
Now that this class is empty, do we need it?

Reply:
@SilasReinagel At the moment we do not have any `Team` specific behaviors, but in the future it'll have the method provided by the `Composite` implementation of the Composite pattern.